terraform-beyond-layer0/example-3/modules/guestbook_service/Dockerrun.aws.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
@zpatrick not sure what you mean here? So the directory structure for `Dockerrun.aws.json` is

```
# current path
/example-3/modules/guestbook_service/Dockerrun.aws.json

# suggested path?
/example-3/modules/guestbook_service/src/Dockerrun.aws.json
```